### PR TITLE
Updating to laravel 5

### PR DESCRIPTION
**Pull request by nWidart**
About #9

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
 	},
 	"require": {
 		"elasticsearch/elasticsearch": "~1.3",
-		"illuminate/support": "~4.2"
+		"illuminate/support": "~5.0"
```
Comment (bramdevries) [inline composer.json]: Better to use `^4|~5` to still support 4.2 of the framework see http://semver.mwl.be/#?package=illuminate%2Fsupport&version=%5E4%7C~5&minimum-stability=stable

Comment (nWidart) [inline composer.json]: Alright, will do that.
I was originally thinking that there would be a tagged version now, with `4.2` and a new tagged version with `5.0`.

**Comment by nWidart** [comment]
I've updated and squashed the commits.

**Comment by Anahkiasen** [comment]
That's not correct, that will pull illuminate/support 4.0 and 4.1 too and they might not work with the package http://semver.mwl.be/#?package=laravel%2Fframework&version=%5E4%7C~5&minimum-stability=stable

**Comment by nWidart** [comment]
Indeed you're right. Updating to use [^4.2|~5](http://semver.mwl.be/#?package=illuminate%2Fsupport&version=%5E4.2%7C~5&minimum-stability=stable)

**Pull request merged**